### PR TITLE
Referrer policy

### PR DESCRIPTION
**Pull request by adanilo**
Changes for Issue #560 plus some manual diffs for referrerpolicy at various places in the spec. that weren't obvious from the issue.

### CODE REVIEW

File: sections/semantics-document-metadata.include [hunk 1]
```diff
@@ -963,6 +974,75 @@
           inappropriate keywords.
         </p>
 
+  : <dfn><code>referrer</code></dfn>
+
+  :: The value must be a <a>referrer policy</a>, which defines the default <a>referrer
+     policy</a> for the <code>Document</code>. [[!REFERRERPOLICY]]
```
Comment (arronei) [inline sections/semantics-document-metadata.include]: Wrapping a link is bad. Could you fix this one link so we don't line break in the middle of an link element.

File: sections/semantics-document-metadata.include [hunk 2]
```diff
@@ -963,6 +974,75 @@
...
+
+     If any <code>meta</code> elements are <a>inserted into the document</a> or
+     <a>removed from the document</a>, or existing <code>meta</code> elements have their <code>name</code>
+     or <code>content</code> attributes changed, user agents must run the following algorithm:
+
+     <ol>
```
Comment (arronei) [inline sections/semantics-document-metadata.include]: Could use the short hand ol list syntax.

File: sections/semantics-document-metadata.include [hunk 3]
```diff
@@ -963,6 +974,75 @@
           inappropriate keywords.
         </p>
 
+  : <dfn><code>referrer</code></dfn>
+
+  :: The value must be a <a>referrer policy</a>, which defines the default <a>referrer
```
Comment (arronei) [inline sections/semantics-document-metadata.include]: Wrapping a link is bad. Could you fix this one link so we don't line break in the middle of an link element.

**Review by arronei — DISMISSED** [review]
I have 2 comments but only the link wrapping one is necessary to correct. Once that is corrected anyone can merge this. The rest of this is fine.

**Comment by adanilo** [comment]
Fixes for review are in https://github.com/w3c/html/pull/954/commits/d9217b67c50203d0d91f9737bd37b89dbf7322e6 - could someone check and merge please?

**Review by chaals — APPROVED** [review]
OK, this seems good to go now - thanks.

**Pull request merged**